cms/templates/container.html

```diff
@@ -8,27 +8,31 @@
 %>
 </%def>
 <%!
+from django.utils.translation import ugettext as _
+
 from contentstore.views.helpers import xblock_studio_url, xblock_type_display_name
 from openedx.core.djangolib.js_utils import (
     dump_js_escaped_json, js_escaped_string
 )
-from openedx.core.djangolib.markup import HTML, ugettext as _
+from openedx.core.djangolib.markup import escape, HTML
```

Comment:
@nedbat prefers "HTML" over "Html", and since he still hasn't gotten over the loss of JSOON, I will happily comply.